packages/flutter/lib/src/material/dialog.dart

```diff
@@ -166,6 +166,7 @@ class AlertDialog extends StatelessWidget {
     this.content,
     this.contentPadding: const EdgeInsets.fromLTRB(24.0, 20.0, 24.0, 24.0),
     this.actions,
+    this.semanticName,
```

Comment:
Not 100% what the native platforms do for an alert, but should this maybe default to "Alert"?

Reply:
done